conditions/conditions_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
@everettraven Have we ci tests using go here? 
Could we also bump 1.18 on the tests only? So that we can ensure that we can run and it works with 1.18?

Reply:
/lgtm cancel

^

Reply:
It can be in a follow up as well 👍 
So, 
/approved

Reply:
I agree, I think it would be best to do this in a follow up when we upgrade to Go 1.18